open3d_conversions/src/open3d_conversions.cpp

```diff
@@ -0,0 +1,93 @@
+#include "open3d_conversions/open3d_conversions.h"
+
+namespace open3d_conversions
+{
+void open3dToRos(const open3d::geometry::PointCloud& pointcloud, sensor_msgs::PointCloud2& ros_pc2, std::string frame_id)
```

Comment:
Typically C++ / ROS / Google styling is 2 spaces. Could you change to that?

Reply:
Done (86b4308)